index.d.ts

```diff
@@ -159,10 +160,11 @@ declare module "react-native-twilio-video-webrtc" {
     };
     enableNetworkQualityReporting?: boolean;
     maintainVideoTrackInBackground?: boolean;
+    videoTrackName?: string;
   };
 
   class TwilioVideo extends React.Component<TwilioVideoProps> {
-    setLocalVideoEnabled: (enabled: boolean) => Promise<boolean>;
+    setLocalVideoEnabled: (enabled: boolean, videoTrackName?: string) => Promise<boolean>;
```

Comment:
I wonder if we need to pass the `videoTrackName` here. Is it possible that we can use what was passed into `connect()` and have the native code save it in a variable?

Reply:
If I remember correctly, there's a creation portion of this function on the native side that had me give this option. 🤔 That said, we can potentially lean on the track set on `connect`. We do need to update the track name in certain scenarios that this may give us the option of enabling without having to pass it into `connect`. One scenario being when we switch cameras.

Reply:
I checked the native code again, and it looks like I left this behind by accident. That said, we probably do need to handle the above scenario. 🤔

Reply:
Yeah, we will have to add support for this later when we want to support two-way-video.  In that case, we'd unpublish the current video track, and then publish a new track with a different name.